cursorless-talon/src/modifiers/simple_scope_modifier.py

```diff
@@ -15,7 +15,23 @@
 )
 def cursorless_simple_scope_modifier(m) -> dict[str, Any]:
     """Containing scope, every scope, etc"""
+    if hasattr(m, "cursorless_simple_scope_modifier"):
+        modifier = m.cursorless_simple_scope_modifier
+
+        if modifier == "every":
+            return {
+                "type": "everyScope",
+                "scopeType": m.cursorless_scope_type,
+            }
+
+        if modifier == "grand":
+            return {
+                "type": "containingScope",
+                "scopeType": m.cursorless_scope_type,
+                "ancestorIndex": 1,
+            }
```

Comment:
Do we not want to support multiple levels? Doesn't seem much harder. Spoken form could be one of:

1. "great grand scope"
2. "grand grand scope"
3. "two grand scope"
4. "second grand scope"

I would lean towards 1) or 2) because 3) and 4) could conflict with modifiers I was thinking we could add in the future where they operate within the grand iteration scope. Eg in the following:

```ts
if (true) {
  const aaa = bbb;
  const ccc = ddd;
}

const eee = fff;
```

saying "second grand state air" would be `const eee = fff;`. I think "every grand state", "next grand state", etc could prob also work. I should file those as an issue. But long story short I'd lean towards "great grand" or "grand grand" to leave our grammatical options open, and because I think repeating the word is fine as you're very unlikely to go beyond 2 I think

The advantage of "grand grand" is that you could implement it super easily I believe: just put `{user.cursorless_modifier_grand}*` in the pattern and then use `ancestor_index = len(m.cursorless_modifier_grand_list)`

Reply:
Maybe we should just start with this one and see how we utilize it?

Reply:
I guess it's consistent with all of our grammar proposals, and appears in the same csv as it would, so I guess it's fine to start here?

Reply:
I think so

Reply:
https://github.com/cursorless-dev/cursorless/issues/2152